dashboard/db/migrate/20170508234535_change_lockable_on_stage.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeLockableOnStage < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :stages, :lockable, false, false
+    change_column_default :stages, :lockable, from: nil, to: false
```

Comment:
It shouldn't matter, since the 4th param of `change_column_null` specifies the default value for NULL entries (`false`).  I believe even if I swap the order, the 4th param is still needed.